Angular InstantSearch/conditional-request/src/app/app.component.ts

```diff
@@ -1,36 +1,45 @@
-import { Component } from '@angular/core';
-import * as algoliasearch from 'algoliasearch/lite';
+import {Component} from '@angular/core';
+import algoliasearch from 'algoliasearch/lite';
+import {MultipleQueriesQuery, MultipleQueriesResponse, SearchResponse} from '@algolia/client-search';
 
-const algoliaClient = algoliasearch(
-  'latency',
-  '6be0576ff61c053d5f9a3225e2a90f76'
+const originalSearchClient = algoliasearch(
+  'B1G2GM9NG0',
+  'aadef574be1f9252bb48d4ea09b5cfe5'
 );
```

Comment:
Why use the docs Algolia app instead of Latency?

The `demo_ecommerce` index was designed for the interactive tutorial and might change in the future, so I'm not sure it's a good idea to rely on it.

Reply:
the doc-code-samples aren't consistent in which app is used, can be done in a separate pr imo

Reply:
👍 Let's just make sure we don't forget to it.